mmv1/templates/terraform/examples/sql_sqlserver_instance_authorized_network.tf.erb

```diff
@@ -0,0 +1,18 @@
+# [START cloud_sql_sqlserver_instance_authorized_network]
+resource "google_sql_database_instance" "<%= ctx[:primary_resource_id] %>" {
+  name             = "<%= ctx[:vars]['myinstance'] %>"
+  region           = "us-central1"
+  database_version = "SQLSERVER_2017_STANDARD"
+  root_password = "INSERT-PASSWORD-HERE"
+  settings {
+    tier = "db-custom-2-7680"
+    ip_configuration {
+      authorized_networks {
+        name = "Network Name"
+        value = "192.0.2.0/24"
+        expiration_time = "3021-11-15T16:19:00.094Z"
+      }
+    }
+  }
+}
```

Comment:
Looks like we need to add `deletion_protection = false` to get tests passed, as the field set to true by default but we need to be able to delete the resource in our tests.

If `deletion_protection = true` needs to be shown in CGC samples, we probably can follow what have been done in this PR (https://github.com/GoogleCloudPlatform/magic-modules/pull/5706), adding `test_vars_overrides`.

Thank you so much!

Reply:
Done